pom.xml

```diff
@@ -324,7 +324,7 @@
     <jackson.version>2.10.5.1</jackson.version>
     <json.version>1.1.1</json.version>
     <jline.version>2.14.6</jline.version>
-    <snappy.version>1.1.7</snappy.version>
+    <snappy.version>1.1.8.4</snappy.version>
```

Comment:
What are the differences between these two versions? We had a couple of issues with snappy versions before, so let's be extremely cautious and do more testing around it :)